app/models/item.rb

```diff
@@ -2,4 +2,5 @@ class Item < ActiveRecord::Base
   validates :name, presence: true, uniqueness: true
   validates :sell_in, presence: true
   validates :quality, presence: true
+  validates :description, presence: true
```

Comment:
Do we want the description to be mandatory or optional? Should there be a character limit in the db to limit how much text should go into the db? Now that this is a required, what will happen when we create a new item in the  current react app?

Reply:
made to allow blank, leaving the default of setting to nil if none given. should not break existing react app